modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/js/util/ShowPartialResultsAlert.tsx

```diff
@@ -27,6 +27,10 @@ const ShowPartialResultsAlert: React.FC<IProps> = ({
 		!showPartialResultsToRespondents
 	);
 
+	const showPartialResultsMessage = dismissible
+		? 'your-responses-will-be-visible-to-all-form-respondents'
+		: 'respondents-can-see-all-submitted-form-data';
+
```

Comment:
Hi @daiazius, as I talked to you on Slack, Liferay.Language.get function doesn't work well with variables. Please, could you test is this way it works?
```suggestion
		const showPartialResultsMessage = dismissible
	? Liferay.Language.get('your-responses-will-be-visible-to-all-form-respondents')
	: Liferay.Language.get('respondents-can-see-all-submitted-form-data');

```